app/views/sessions/_form.html.erb

```diff
@@ -9,7 +9,7 @@
 
             <div class='form-spacing'></div>
 
-<%= form_tag '/login' do %>
+<%= form_tag '/sessions' do %>
```

Comment:
@mattchang92 also fixed this and has a capybara test for it. Favor his solution when resolving the conflict